app/controllers/api/v1/facts_controller.rb

```diff
@@ -0,0 +1,76 @@
+class Api::V1::FactsController < ApplicationController
+    include AuthenticationCheck
+  
+    before_action :is_user_logged_in
+    before_action :set_fact, only: [:show, :update, :destroy]
+    before_action :check_access
+  
+    # GET /members/:member_id/facts
+    def index
+      @member = Member.find(params[:member_id])
```

Comment:
I know that `check_access` is perhaps not the best name for something that does both a check and an assignment to `@member`, but because `@member` is already set in a `before_action`, you don't need it here. Same applies to some of the `create` method below.